### PR TITLE
Remove twilio 

### PR DESCRIPTION
**Pull request by xenein**
this work in progress removes the remaining twilio-parts as their api is discontinued.﻿

### CODE REVIEW

**Comment by xenein** [comment]
this should remove all parts of the code that use parts of the twilio package or API.

utils.py uses `TWILIO_ACCOUNT_SID` to HMAC and verify. 

- what to do with that? Change to `TELNYX_APP_ID`?
- by docstring the "HMAC" is the hash of `$first_component:secret` (where first component is a base64 representation of the json-dump of the to be signed object.
  -  This is not how HMACs usually are composed, usually the part with the secret goes in first and there is multiple rounds of hashing. This is done because appending the secret allows for some cheap attack based upon the sponginess of SHA1 and friends.
  - all the methods are named something with `sign` - which would imply that there's asymmetric cryptography involved. The same secret is used for verifying and "signing" so this is naming seems a bit off, too.

File: froide_fax/utils.py [hunk 1]
```diff
@@ -52,36 +50,27 @@ def get_signature(user):
...
-FAX_CALLBACK_SALT = 'fax_callback_url'
+FAX_MEDIA_SALT = "fax_media_url"
+FAX_CALLBACK_SALT = "fax_callback_url"
 
 
 def get_media_url(att):
-    # Telnyx doesn't seem to follow redirects
-    # Give short-lived direct authenticated URL
-    # instead of permanent signed URL that redirects
-    return att.get_absolute_domain_file_url(authorized=True)
```
Comment (stefanw) [inline froide_fax/utils.py]: This change here seems to have been lost, but it's essential for Telnyx integration to work.

**Review by stefanw — CHANGES_REQUESTED** [review]
This commit 3a5b67ddfa9de058ccda5f8f1fa7ed0bee56b2c4 seems to have not made it into this PR.

File: froide_fax/utils.py (hunk 1 above)
Comment (xenein) [reply froide_fax/utils.py]: Oh. I thought I started with a fresh pull on the master-branch. I'll look into that.

Comment (xenein) [reply froide_fax/utils.py]: I cherry-picked that one back in.

**Pull request merged**